pages/search/_query/index.vue

```diff
@@ -39,10 +19,6 @@ export default {
 	},
 	computed: {
 		searchQuery () {
-			// const myRegexp = /rch\?s=(.*)/
-			// const currentPath = this.$nuxt.$route.fullPath
-			// const match = myRegexp.exec(currentPath)
-			// return match[1]
 			return decodeURIComponent(this.$nuxt.$route.fullPath.match(/rch\?s=(.*)/)[1])
 		},
 		filterdBlogs () {
```

Comment:
I suggest renaming this to `foundPosts`.

Reply:
Also, rename `filteredPages` to `foundPages`, and rename `pagesPosts` to `searchResults`.

Reply:
Also, remove `groupedPosts` and the `lodash` import as it's not being used.